dipy/direction/__init__.py

```diff
@@ -4,3 +4,4 @@
 from .probabilistic_direction_getter import DeterministicMaximumDirectionGetter
 from .ptt_direction_getter import PTTDirectionGetter
 from .peaks import *
+from .bingham import *
```

Comment:
`import *` to avoid. Please import only what you think necessary. 

I will create a PR to remove the one above (`.peaks import *`). I just saw it and this is the only one in the codebase